### DISCUSSION TITLE
请问作者可以再出9x漫画规则吗

### DISCUSSION THREAD

**Discussion by 13316282905** [xiaohucode/yidaRule#103] [Ideas]
请作者再出9x漫画规则。出更多的漫画规则

**Comment by xiaohucode** [comment]
自己写

**Reply by 13316282905** [reply]
我不会写，请问可以出写规则的视频吗

**Comment by a1850976305** [comment]
大佬那个推特规则不能用，能修一下吗？

**Reply by xiaohucode** [reply]
需要登录自己的账号。在发现页 右上角登录

**Comment by 13316282905** [comment]
![Screenshot_2024-03-11-18-14-09-715_com miui home](https://github.com/xiaohucode/yidaRule/assets/159050612/4f1d105f-8210-4a80-8709-b536335e5ee9)
请问作者可以添加一下这个9x阅读器吗
